cmake/PegasusConfig.cmake

```diff
@@ -10,9 +10,18 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH "${PEGASUS_ROOT}/Pegasus/cmake")
 
-include(GlmConfig)
+#Build flags
+if (UNIX)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Werror -pedantic")
```

Comment:
I assume this code was copied `Pegasus/CMakeLists.txt`, and it had `-fno-exceptions` flag. It's not present here. Intentional?

Reply:
@yanenkoa Yes. It is intentional.